packages/core-webhooks/lib/manager.js

```diff
@@ -50,23 +50,24 @@ class WebhookManager {
     const matches = []
 
     for (const webhook of webhooks) {
-      if (!webhook.conditions) {
-        matches.push(webhook)
+      if (webhook.enabled == 1) {
```

Comment:
Also, instead of wrapping the whole block into an `if` you could just do an early exit.

```
if(!webhook.enabled) {
  continue;
}
```

Reply:
done, this is simpler. Updated code with change